app/lib/account/session_cookie.dart

```diff
@@ -103,15 +136,32 @@ Map<String, Object> clearSessionCookies() {
         maxAge: Duration.zero,
       ),
       buildSetCookieValue(
-        name: _clientSessionLaxCookieName,
+        name: clientSessionLaxCookieName,
         value: '',
         maxAge: Duration.zero,
       ),
       buildSetCookieValue(
-        name: _clientSessionStrictCookieName,
+        name: clientSessionStrictCookieName,
         value: '',
         maxAge: Duration.zero,
       ),
     ],
   };
 }
+
+/// The session cookies present with the request, with the current session identifier.
+class ClientSessionCookieStatus {
+  final bool isPresent;
+  final bool isStrict;
+  final String? sessionId;
+  final bool needsReset;
```

Comment:
Documentation comments, if we really want to carry around this much information.

If `needsReset` is true, then wouldn't it almost always be okay to throw a `ResponseException` that clears cookies?